tests/integration/goldens/credentials/samples/generated_samples/iamcredentials_generated_credentials_v1_iam_credentials_generate_access_token_async.py

```diff
@@ -35,6 +35,8 @@ async def sample_generate_access_token():
 
     # Initialize request argument(s)
     request = credentials_v1.GenerateAccessTokenRequest(
+        name="projects/{project}/serviceAccounts/{service_account}",
+        scope=['scope_value'],
```

Comment:
if the scope is of type list is it worth adding more than one list item?

Reply:
That sounds reasonable and should be an easy fix - opened https://github.com/googleapis/gapic-generator-python/issues/961